src/mmw/apps/user/views.py

```diff
@@ -93,10 +93,36 @@ def itsi_auth(request):
     else:
         # User did not authenticate. Save their ITSI ID and send to /register
         request.session['itsi_id'] = itsi_user['id']
-        return redirect('/register')  # TODO Make this Backbone route
+        return redirect('/register/{username}/{first_name}/{last_name}'.format(
+            **itsi_user['extra']
```

Comment:
Yeah. We get these values from ITSI and we'd like to populate them in the form on the client side. We can either send this via the URL, or cache it in the server and have the client do a second AJAX request, or set a cookie and have the client read that and destroy it once its done. This seemed like the easiest way to do it.